WordPress/src/main/res/layout/subfilter_page_fragment.xml

```diff
@@ -13,7 +13,6 @@
         android:clipToPadding="false"
         android:descendantFocusability="beforeDescendants"
         android:scrollbars="vertical"
-        android:layout_marginTop="@dimen/margin_extra_large"
         android:layout_marginBottom="@dimen/margin_extra_large"/>
```

Comment:
You might want to switch from `layout_marginBottom` to `paddingBottom` here, so that the `clipToPadding` has effect and the list won't cut-off the items at the bottom.

Reply:
Good point 👍! Changed it here 6347458